### ISSUE TITLE
 a working gpgtools 7 beta and sonoma

### ISSUE THREAD

**Issue by decorte** [Free-GPGMail/Free-GPGMail#100]
hey there, I just installed the functional gpg tools beta for Sonoma. is there a way to follow the new architecture with free-GPGmail?
thanks

**Comment by decorte** [comment]
beta binary is published here:
https://gpgtools.tenderapp.com/discussions/feedback/17913-got-your-email-about-sonoma-support

**Comment by bnavigator** [comment]
That is good news! I don't have the resources to test it out myself.

Did they also publish the source code, yet? Does the dmg package come with an open source license?

**Comment by decorte** [comment]
the code should be published in the beta branch, don't know
https://github.com/GPGTools/GPGMail/tree/beta

the architecture changed with sonoma mac mail update (move to extensions) so i wouldn't even know how to test it...

**Comment by bnavigator** [comment]
Nah, they didn't update their public github repository in years.

**Comment by john-nethemba** [comment]
The DMG for the second beta (2024.1b2) just links to https://gpgtools.org/legal 

> GPG Suite is a software product developed and distributed by GPGTools GmbH, a limited liability company established and existing under the laws of Austria.
> 
> By continuing with the installation of GPG Suite you confirm that you are agreeing to be bound to our terms of distribution.
> 
> Terms of Distribution: [https://gpgtools.com/legal](https://gpgtools.org/legal)

 which does show GPLv3 for plugin. Do not know if it will change on actual v8 drop.

**Comment by decorte** [comment]
So, they published second, public beta (2024.1b2) and I see there was some activity on their github but as far as I am able to understand, they only updated Version.config file there. Not sure if this helps.
https://gpgtools.org/sonoma
https://github.com/GPGTools/GPGTools_Installer/commit/28e22e90ccf47ddd74165275be8ff36fd42905ee

**Comment by Billito00** [comment]
Is there any news about Free-GPGMail with MacOS14?

**Comment by bnavigator** [comment]
Unfortunately not. Not-so-free GPGMail is still "beta" and they still don't comply with the GPL by releasing the source code.

I recommend to use Thunderbird for Mac instead.

**Comment by Cyb3rrunn3r** [comment]
My problem with Thunderbird is that i can not get gpg to work with private key on a yubikey and ONLY there! 😔
With enigmail long time ago it worked. 😔

**Comment by nishogi** [comment]
Is there any chance that there is going to be a working version for Sonoma in the future ?

**Comment by austinhuang0131** [comment]
https://infosec.exchange/@GPGTools/114462911696625592

So apparently they have nightlies and on the page there's the wording "source code on request"

So could anyone try to ask for source code that way?

**Comment by bnavigator** [comment]
Try it.

But do you really want to work with code that has been in beta for years? Mind you we are talking about software deeply involved with security and privacy matters. I wouldn't trust a software vendor with such a sloppy handle on code quality, their communication strategy and without proper respect for open source licensing manners. Use a proper client instead.

**Comment by octavian2204** [comment]
Yep, give an example of “proper client”? I was using Thunderbird, which has it built in but it was draining my M1 Mac drastically. I don’t if it was general issue or only on my Mac but with thunderbird 4h of usage, without it - 7-8. In a final, bought GPG tools and stay happy without updating it 3rd year.

**Comment by lordp1** [comment]
maybe this project is a good replacement? https://github.com/mahaupt/mailgpg

**Comment by decorte** [comment]
> maybe this project is a good replacement? https://github.com/mahaupt/mailgpg

this works real fine, thanks!
